crates/starknet-devnet/tests/get_transaction_by_hash.rs

```diff
@@ -42,7 +42,7 @@ mod get_transaction_by_hash_integration_tests {
             &devnet.json_rpc_client,
             signer,
             account_address,
-            chain_id::TESTNET,
+            chain_id::SEPOLIA,
```

Comment:
although its done by another PR, why not change it to use constants::CHAIN_ID

Reply:
what other PR?

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/396/commits/afc3ae3ac3d0faf89a41f402528504934dc5833e